api/src/main/java/com/datastrato/gravitino/authorization/Privilege.java

```diff
@@ -94,7 +94,24 @@ enum Name {
     /** The privilege to revoke a role from the user or the group. */
     REVOKE_ROLE(0L, 1L << 32),
     /** The privilege to get a role */
-    GET_ROLE(0L, 1L << 33);
+    GET_ROLE(0L, 1L << 33),
+
+    TABULAR_SELECT(0L, 1L << 34),
+    TABULAR_UPDATE(0L, 1L << 35),
+    TABULAR_CREATE(0L, 1L << 36),
+    TABULAR_DROP(0L, 1L << 37),
+    TABULAR_ALTER(0L, 1L << 38),
+    TABULAR_INDEX(0L, 1L << 39),
+    TABULAR_LOCK(0L, 1L << 40),
+    TABULAR_READ(0L, 1L << 41),
+    TABULAR_WRITE(0L, 1L << 42),
+    TABULAR_REPL_ADMIN(0L, 1L << 43),
+    TABULAR_SERVICE_ADMIN(0L, 1L << 44),
+    /** Temporary UDF Admin */
+    TABULAR_TEMP_UDF_ADMIN(0L, 1L << 45),
+    TABULAR_REFRESH(0L, 1L << 46),
+    TABULAR_ALL(0L, 1L << 47),
```

Comment:
What if the enum instance is more than 64?

Reply:
The enum `Name` type has `highBits` and `lowBits`, if `lowBits` more than 64, we can use `highBits`.